RS.Fritz.Manager.API/Capture/CaptureControlService.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace RS.Fritz.Manager.API;
+
+using System;
+using System.Globalization;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+internal sealed class CaptureControlService : ICaptureControlService
+{
+    private readonly IHttpClientFactory httpClientFactory;
+
+    public CaptureControlService(IHttpClientFactory httpClientFactory)
+    {
+        this.httpClientFactory = httpClientFactory;
+    }
+
+    public async Task GetStartCaptureResponseAsync(Uri uri, string folderPath, string filePrefix)
+    {
+        HttpClient httpClient = httpClientFactory.CreateClient(Constants.HttpClientName);
+        var file = new FileInfo(FormattableString.Invariant($"{folderPath}\\{filePrefix}_{DateTime.Now.ToString("dd/MM/yyyy'_'HH'_'mm'.'ss", CultureInfo.CreateSpecificCulture("de"))}.eth"));
```

Comment:
I suggest to use [ISO8601](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#Sortable) format here using:

```C#
DateTime.Now.ToString("s").Replace(":", string.Empty)
```